crates/bevy_gizmos/src/lib.rs

```diff
@@ -269,10 +289,68 @@ struct LineGizmoHandles {
     strip: Option<Handle<LineGizmo>>,
 }
 
+/// Stash the default gizmos context into the gizmo storage swap buffer.
+pub fn stash_default_gizmos(world: &mut World) {
+    world.resource_scope(|world: &mut World, mut swap: Mut<GizmoStorage<Swap>>| {
+        let mut default = world.resource_mut::<GizmoStorage<()>>();
+        default.swap(&mut *swap);
+    });
+}
```

Comment:
Why use an exclusive system here when one can do
```suggestion
pub fn stash_default_gizmos(
    mut swap: ResMut<GizmoStorage<Swap>>,
    mut default: ResMut<GizmoStorage<()>>,
) {
    default.swap(&mut *swap);
}
```

Reply:
Just some residual stuff that I forgot about from a different implementation.